torch_xla/distributed/xla_dist.py

```diff
@@ -247,11 +250,47 @@ def _docker_run_cmd(self, cmd):
     docker_cmd.extend(cmd)
     return docker_cmd
 
+  def _tpuvm_env_vars_cmd(self, worker_idx):
+    env_vars = {
+        xenv.TPU_CHIPS_PER_HOST_BOUNDS: '2,2,1',
+        xenv.TPUVM_MODE: 1,
+        xenv.CLOUD_TPU_TASK_ID: worker_idx,
+    }
+    accelerator_type = self._cluster.get_service_workers()[0]._machine_type
```

Comment:
`get_service_workers` is renamed so this shouldn't work?

Reply:
Ah.. the function I renamed is `ClusterResolver. get_service_workers `. We also have a `Cluster.get_service_workers` so this function still works.